Gemfile

```diff
@@ -11,11 +11,18 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
 # Use SCSS for stylesheets
 gem 'sass-rails', '~> 5.0'
+gem 'jquery-rails'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'mini_racer', platforms: :ruby
-
+gem 'bootstrap', '~> 4.1.1'
+gem 'kaminari'
+gem 'bootstrap-select-rails'
+gem 'selectize-rails'
+gem 'pry-rails'
```

Comment:
You probably should move this one to dev and test envs, you don't need this gem in production.